app/src/main/scala/org/alephium/explorer/persistence/model/ContractEntity.scala

```diff
@@ -16,14 +16,17 @@
 
 package org.alephium.explorer.persistence.model
 
-import org.alephium.api.model.ValAddress
+import akka.util.ByteString
+
+import org.alephium.api.model.{ValAddress, ValByteVec}
 import org.alephium.protocol
 import org.alephium.protocol.model.{Address, BlockHash, TransactionId}
 import org.alephium.util.TimeStamp
 
 final case class ContractEntity(
     contract: Address,
     parent: Option[Address],
+    interfaceId: Option[ByteString],
```

Comment:
Should we name it explicitly that the id is just a guess and the id is for standard interface? Something like `stdInterfaceIdGuessed`.